competitive coding 3/pascals triangle.txt

```diff
@@ -0,0 +1,27 @@
+
+class Solution {

```

Comment:
read the question properly
and submit the solutions of leetcode first before submitting the pull requests